src/main/groovy/wooga/gradle/build/unity/ios/IOSBuildPlugin.groovy

```diff
@@ -388,4 +394,43 @@ class IOSBuildPlugin implements Plugin<Project> {
         archiveDSYM.configure({ it.mustRunAfter(xcodeExport) })
         tasks.named(BasePlugin.ASSEMBLE_TASK_NAME).configure({ it.dependsOn(xcodeExport, archiveDSYM, collectOutputs) })
     }
+
+    /***
+     * Applies the asdf plugin, which manages the installation and usage of the asdf tool which manages
+     * multiple language runtime versions on a per-project basis.
+     * (https://github.com/asdf-vm/asdf)
+     * @param project
+     */
+    void applyAsdfPlugin(Project project) {
+
+        /**
```

Comment:
Can you please remove those comments. I understand why they were here, but they aren't needed anymore.

Reply:
Sure.